Android/shreyshrivastava/Advanced/NewsFeed/app/build/intermediates/dex_number_of_buckets_file/debug/out

```diff
@@ -0,0 +1 @@
+2
```

Comment:
Avoid uploading app/build files into the repository.

Reply:
Okay.